src/Libraries/PythonNodeModels/PythonNode.cs

```diff
@@ -83,13 +83,26 @@ protected override string GetInputTooltip(int index)
             var vals = additionalBindings.Select(x => x.Item2).ToList();
             vals.Add(AstFactory.BuildExprList(inputAstNodes));
 
-            Func<string, IList, IList, object> backendMethod =
-                IronPythonEvaluator.EvaluateIronPythonScript;
+            Func<string, IList, IList, object> pythonEvaluatorMethod;
+            if (Engine == PythonEngineVersion.CPython3)
+            {
+                // TODO: Uncomment this and replace the following code when merging Python3 changes in master
+                // pythonEvaluatorMethod = DSCPython.CPythonEvaluator.EvaluatePythonScript;
```

Comment:
It would be nice to add this in the AC of whatever task will merge the Python3 branch into master. Otherwise we may forget to do this.